packages/editor-ui/src/views/CredentialsView.vue

```diff
@@ -0,0 +1,430 @@
+															:label="user.fullName"
+														>
+															<n8n-user-info
+																v-bind="user"
+																:isCurrentUser="user.id === currentUser.id"
+																:class="$style['user-info']"
+															/>
+														</n8n-option>
+													</template>
+												</n8n-select>
+											</enterprise-edition>
+											<div :class="[$style['filters-dropdown-footer'], 'mt-s']">
+												<n8n-link @click="resetFilters" v-if="hasFilters">
+													{{ $locale.baseText('credentials.filters.reset') }}
+												</n8n-link>
+											</div>
+										</div>
+									</el-dropdown-menu>
+								</el-dropdown>
+							</div>
+						</div>
+					</div>
+				</template>
+
+				<div v-show="hasFilters" class="mt-xs">
+					<n8n-info-tip :bold="false">
+						{{ $locale.baseText('credentials.filters.active') }}
+						<n8n-link @click="resetFilters" size="small">
+							{{ $locale.baseText('credentials.filters.active.reset') }}
+						</n8n-link>
+					</n8n-info-tip>
+				</div>
+
+				<div class="mt-xs mb-l">
+					<ul class="list-style-none" v-if="filteredAndSortedCredentials.length > 0">
+						<li v-for="credential in filteredAndSortedCredentials" :key="credential.id" class="mb-2xs">
+							<credential-card :data="credential" />
+						</li>
+					</ul>
+					<n8n-text color="text-base" size="medium" v-else>
+						{{ $locale.baseText('credentials.noResults') }}
+					</n8n-text>
+				</div>
+			</page-view-layout-list>
+		</template>
+	</page-view-layout>
+</template>
+
+<script lang="ts">
+import { showMessage } from '@/components/mixins/showMessage';
+import {ICredentialsResponse} from '@/Interface';
+import mixins from 'vue-typed-mixins';
+
+import SettingsView from './SettingsView.vue';
+import PageViewLayout from "@/components/layouts/PageViewLayout.vue";
+import PageViewLayoutList from "@/components/layouts/PageViewLayoutList.vue";
+import CredentialCard from "@/components/CredentialCard.vue";
+import {CREDENTIAL_SELECT_MODAL_KEY} from "@/constants";
+import {ICredentialType} from "n8n-workflow";
+import {EnterpriseEditionFeature} from "@/constants";
+import {mapGetters} from "vuex";
+import TemplateCard from "@/components/TemplateCard.vue";
+import Vue from "vue";
+
+export default mixins(
+	showMessage,
+).extend({
+	name: 'SettingsPersonalView',
+	components: {
+		TemplateCard,
+		PageViewLayout,
+		PageViewLayoutList,
+		SettingsView,
+		CredentialCard,
+	},
+	data() {
+		return {
+			loading: true,
+			filters: {
+				owner: true,
+				sortBy: 'lastUpdated',
+				search: '',
+				type: [] as string[],
+				ownedBy: '',
+				sharedWith: '',
+			},
+			filtersInput: {
+				type: [] as string[],
+				ownedBy: '',
+				sharedWith: '',
+			},
+			EnterpriseEditionFeature,
+		};
+	},
+	computed: {
+		...mapGetters('users', ['currentUser', 'allUsers']),
+		...mapGetters('credentials', ['allCredentials', 'allCredentialTypes', 'credentialTypesById']),
```

Comment:
nitpick - I have come to hate `mapGetters` because it does not set types.. setting each one alone is annoying but types are helpful

Reply:
Migrated to computed functions.